features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Applications/Details/InfoBar.jsx

```diff
@@ -234,7 +233,9 @@ class InfoBar extends React.Component {
         if (typeof todo === 'boolean') {
             this.setState({ showOverview: todo });
         } else {
-            this.setState({ showOverview: !this.state.showOverview });
+            this.setState((prevState) => {
```

Comment:
You could just omit the return keyword , if you unwrap the method body from curlybraces 
i:e
```javascript
            this.setState( prevState => ( { showOverview: !prevState.showOverview } ));
```

Reply:
fixed  https://github.com/wso2/carbon-apimgt/pull/6787/commits/b09339ab96cfdcc35f13e351833f406efad26ead